.storybook/stories/datagrid/datagrid-expandable-row.stories.ts

```diff
@@ -14,6 +14,12 @@ import { setupStorybook } from '../../helpers/setup-storybook.helpers';
 
 const defaultStory: Story = args => ({
   template: `
+  <style>
+  .open-tooltip {
+    visibility: visible;
+    opacity: 1;
+  }
+  </style>
```

Comment:
```suggestion
  <style>
    .open-tooltip {
      visibility: visible;
      opacity: 1;
    }
  </style>
```

still missing indentation